test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Models/ListObjectModel.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp.Models
+{
+    public class ListObjectModel<T>
+    {
+        public List<T> List { get; set; }
```

Comment:
The name, `List` is not correct. If the name of `T` is `BlackWidows`, the property name of `List<T>` should also be `BlackWidows`, not `List`.